my_main.py

```diff
@@ -0,0 +1,67 @@
+# HMMMM what should i do
+# I think book-reading should be an easy one. Or a math computation of some kind.
+# Inputs: number
```

Comment:
The inputs and outputs need of each function need to be described in a docstring beneath def function()